src/rgw/rgw_sal.h

```diff
@@ -358,7 +358,9 @@ class Store {
     /** Enable or disable a set of bucket.  e.g. if a User is suspended */
     virtual int set_buckets_enabled(const DoutPrefixProvider* dpp, std::vector<rgw_bucket>& buckets, bool enabled) = 0;
     /** Get a new request ID */
-    virtual uint64_t get_new_req_id() = 0;
+    virtual uint64_t get_new_req_id() {
+      return ceph::util::generate_random_number<uint64_t>();
```

Comment:
it's not clear how rgw_sal.h is getting this "include/random.h" include - can you please #include it explicitly?